readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md

```diff
@@ -0,0 +1,212 @@
+﻿# Opensds Telemetry with LVM: User Guide
+
+
+## Getting Started!
+This guide is a "Hello World"-style tutorial which shows how to install, configure, and use Telemetry feature in a simple opensds with lvm  setup.
+By following this guide you will be able to install opensds with telemetry tools, create test volumes, create sample grafana dashboard, create sample alert rules and view dashboards.
+
+## Deploying Opensds with Telemetry
+Follow this opensds wiki link to install opensds with telemetry and 
+***Pay special attention to the section below before starting the deploy operation!***
+
+**Download opensds-installer code**
+
+    ```
+    
+     git clone https://github.com/opensds/opensds-installer.git
+    cd opensds-installer/ansible
+    # Checkout the latest release, give master by defualt or specfy the branch 
+    git checkout master
+    ```
+    
+  **How to enable Telemetry installation**
+  
+   Update the file : ansible/group_vars/telemetry.yml for install_telemetry_tools
+   
+   
+    
+    ```
+    # Do you need to install telemetry tools?
+    install_telemetry_tools: true
```

Comment:
s/install_telemetry_tools/enable_telemetry_tools

Reply:
modified text, config present is /ansible/groupvars/telemetry.yml

Reply:
It's still not changed.

Reply:
This is the text as in install script.We just mention it here ofr user awareness.   Are you asking to change the field name in install scripts?

Reply:
@leonwanghui Please see my comment above. Let me know if you have any quesion.